### DISCUSSION TITLE
Didn't receive wFTT in my sollet

### DISCUSSION THREAD

**Discussion by AltcoinHuckster** [wormhole-foundation/wormhole#238] [Problems / bugs] [answered]
Hi,

I bridged ERC20 FTT (https://etherscan.io/tx/0x8f326e83d940b07b87f4cd5da731d30e0c1f057244120f376cf75db53ad4d43e) using https://www.wormholebridge.com/ and the wFTT didn't appear on my sollet address (Ei3S5HvitmqTj2Fv3uDnvYpyjy8fcpdyvDeHKJCA7Dqt)

Please kindly assist.

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/3RMRMrbGHqV2C93TcZx2kXqkuo3HPNwB8cDgcoWuSHphC2TsQZgZfbrWvw2QK5HAZbSwAGqkZ4v4XQPFhbJogEAE
